File: javascript 90 questions/Q6.js

function check_login(username,password){
    const admin = "admin"
    const correct_password = 1234
    if (username === admin && password === correct_password){
        console.log("successful")
    }else{
        console.log("login failed")
    }

}
check_login("admin",1234);
check_login("admin",1245);

File: C questions/Q6.c
#include <stdio.h>
#include <string.h> 

int main() {
    char admin[] = "admin";  
    int correct_password = 1234; 
    char username[20];  
    int password;  

    printf("Enter username: ");
    scanf("%s", username);
    printf("Enter password: ");
    scanf("%d", &password);
    if (strcmp(admin, username) == 0 && password == correct_password) {
        printf("Login successful\n");
    } else {
        printf("Login failed\n");
    }
    return 0;
}

